x-pack/plugins/ingest_pipelines/public/index.ts

```diff
@@ -13,3 +13,4 @@ export function plugin() {
 
 export { INGEST_PIPELINES_APP_LOCATOR, INGEST_PIPELINES_PAGES } from './locator';
 export type { IngestPipelinesListParams } from './locator';
+export type { Pipeline } from '../common/types';
```

Comment:
I don't think we need this anymore, we're returning the pipelines type directly from the elasticsearch client and using that type, so we can remove this change.

Reply:
Updated